From 8ea9ccd75ebf0d69bf08a8be85d829e1615a8bb2 Mon Sep 17 00:00:00 2001
From: Olaf Hering <olaf@aepfle.de>
Date: Fri, 18 Dec 2020 22:28:38 +0100
Subject: Fix missing-prototypes

model_user.c:395:1: error: no previous prototype for 'places_bookmarks_user_create' [-Werror=missing-prototypes]
model_volumes.c:572:1: error: no previous prototype for 'places_bookmarks_volumes_create' [-Werror=missing-prototypes]
model.c:141:1: error: no previous prototype for 'places_bookmark_group_create' [-Werror=missing-prototypes]
model_system.c:267:1: error: no previous prototype for 'places_bookmarks_system_create' [-Werror=missing-prototypes]
button.c:85:29: error: no previous prototype for 'places_button_get_type' [-Werror=missing-prototypes]
button.c:281:1: error: no previous prototype for 'places_button_new' [-Werror=missing-prototypes]

Signed-off-by: Olaf Hering <olaf@aepfle.de>
---
 panel-plugin/button.h        | 4 ++--
 panel-plugin/model.h         | 2 +-
 panel-plugin/model_system.h  | 2 +-
 panel-plugin/model_user.h    | 2 +-
 panel-plugin/model_volumes.h | 2 +-
 5 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/panel-plugin/button.h b/panel-plugin/button.h
index f8ebd20..f6e3f30 100644
--- a/panel-plugin/button.h
+++ b/panel-plugin/button.h
@@ -60,10 +60,10 @@ struct _PlacesButtonClass
 };
 
 GType
-places_button_get_type();
+places_button_get_type(void);
 
 GtkWidget*
-places_button_new();
+places_button_new(XfcePanelPlugin *plugin);
 
 void
 places_button_set_label(PlacesButton*, const gchar *label);
diff --git a/panel-plugin/model.h b/panel-plugin/model.h
index d23e9e6..c3b4bc5 100644
--- a/panel-plugin/model.h
+++ b/panel-plugin/model.h
@@ -85,7 +85,7 @@ GList* places_bookmark_group_get_bookmarks(PlacesBookmarkGroup*);
 
 gboolean places_bookmark_group_changed(PlacesBookmarkGroup*);
 
-PlacesBookmarkGroup* places_bookmark_group_create();
+PlacesBookmarkGroup* places_bookmark_group_create(void);
 
 void places_bookmark_group_destroy(PlacesBookmarkGroup*);
 
diff --git a/panel-plugin/model_system.h b/panel-plugin/model_system.h
index 48023ca..5938ba3 100644
--- a/panel-plugin/model_system.h
+++ b/panel-plugin/model_system.h
@@ -25,7 +25,7 @@
 #include "model.h"
 
 PlacesBookmarkGroup*
-places_bookmarks_system_create();
+places_bookmarks_system_create(void);
 
 #endif
 /* vim: set ai et tabstop=4: */
diff --git a/panel-plugin/model_user.h b/panel-plugin/model_user.h
index fba8766..a114d7e 100644
--- a/panel-plugin/model_user.h
+++ b/panel-plugin/model_user.h
@@ -25,7 +25,7 @@
 #include "model.h"
 
 PlacesBookmarkGroup*
-places_bookmarks_user_create();
+places_bookmarks_user_create(void);
 
 #endif
 /* vim: set ai et tabstop=4: */
diff --git a/panel-plugin/model_volumes.h b/panel-plugin/model_volumes.h
index 2af804a..50aebc7 100644
--- a/panel-plugin/model_volumes.h
+++ b/panel-plugin/model_volumes.h
@@ -25,7 +25,7 @@
 #include "model.h"
 
 PlacesBookmarkGroup*
-places_bookmarks_volumes_create();
+places_bookmarks_volumes_create(gboolean mount_and_open_by_default);
 
 #endif
 /* vim: set ai et tabstop=4: */
-- 
cgit v1.2.3

